sdk/communication/communication-chat/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 1.4.0 (TBD)
+
+- Added inline image support
+
```

Comment:
Need to add `- Updated @azure/communication-signaling to 1.0.0-beta.22 with inline image support.`